e2e/tshark/tshark_test.go

```diff
@@ -0,0 +1,82 @@
+package basic
+
+import (
+	"context"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+func TestTshark(t *testing.T) {
+	t.Parallel()
+	// Setup
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	knuu, err := knuu.New(ctx)
+	if err != nil {
+		t.Fatalf("Error creating knuu: %v", err)
+	}
```

Comment:
instead of this I suggest to use `require.NoError(t, err, "optional text")` it is more concise.